.gitignore

```diff
@@ -44,6 +44,7 @@ verilator_coverage_bin*
 /.vscode/
 /.idea/
 /cmake-build-*/
+/nodist/coverage/
```

Comment:
obj_dir/coverage

or obj_coverage

Something that doesn't hide in what looks like source directory and `rm -rf obj*` will remove.